test/TesterInternal/MembershipTests/MembershipTableTestsBase.cs

```diff
@@ -114,23 +114,28 @@ protected virtual string GetAdoInvariant()
             Assert.Equal(0, data.Version.Version);
         }
 
-        protected async Task MembershipTable_InsertRow()
+        protected async Task MembershipTable_InsertRow(bool extendedProtocol = true)
         {
             var membershipEntry = CreateMembershipEntryForTest();
 
             var data = await membershipTable.ReadAll();
             Assert.NotNull(data);
             Assert.Equal(0, data.Members.Count);
 
-            bool ok = await membershipTable.InsertRow(membershipEntry, data.Version.Next());
+            TableVersion nextTableVersion = data.Version.Next();
```

Comment:
I think this line was added by mistake. Please run the tests on Azure to make sure you haven't mistakenly broke anything related to the extended protocol.